growthstreet_zoo/zoo.py

```diff
@@ -117,17 +60,31 @@ def __str__(self):
 
     def __repr__(self):
         """Class representation, returned when class object is called."""
-        return '<%s: %s (%s)>' % (self.__class__.__name__, self.name, self.cage_contents)
+        return '<%s: %s (%s)>' % (self.__class__.__name__, self.name, self.contents)
 
     def add_animal(self, animal):
-        animal.get_competition_score()
-        animal.cage = self
-        self.cage_contents.append(animal)
+        """Add an instance of an animal to a Cage."""
+        self.contents.append(animal)
```

Comment:
Nice.  I prefer this approach to the add_cage API above which is slightly different.